src/locales/en.json

```diff
@@ -290,5 +292,8 @@
       "title": "Public"
     },
     "title": "Board visibility"
+  },
+  "all-events": {
+    "title": "All events"
   }
```

Comment:
Please sort properties alphabetically (action available in WebStorm quick action menu)